lib/better_radar/configuration.rb

```diff
@@ -1,6 +1,6 @@
 class BetterRadar::Configuration
 
-  attr_accessor :feed_name, :delete_after_transfer, :username, :key, :language, :log_filename, :log_path
+  attr_accessor :feed_name, :delete_after_transfer, :username, :key, :language, :log_filename, :log_path, :only_recent
```

Comment:
It would be idiomatic to expose a trivial predicate for the reader part of this newly introduced accessor.